src/components/welcomeScreen/WelcomeScreen.tsx

```diff
@@ -0,0 +1,17 @@
+import { Center } from "./WelcomeScreen.Center";
```

Comment:
`Center` can be the default export instead.
Also, do we want to use `.` in the file name?

Reply:
One worry I have is the component is not nested inside one parent - meaning all welcome screen components should be rendered inside one wrapper welcome screen (similar to the way how its showing in jsx) and should have a top level class `"welcomeScreen` as well which I don't see in the DOM.
Right now the children are getting distributed in different parents, since the children are absolutely positioned can we wrap them inside same parent div ? And in that case here we will have a `div` with class `welcomeScreen` which will render all the children

Bascially this is what I am talking about 👇🏻 

![image](https://user-images.githubusercontent.com/11256141/212004619-12203fb8-469c-498f-beee-6ddcb543f6a8.png)

And this is not semantically correct as children are spread across and not a single component

Reply:
Correct, as I pointed out in the description.

I don't see it as a problem. It's an implementation detail and has no bearing on the public API (unless we decided to render welcome screen components separately).

Right now, wrapping it inside a single DOM element would not be easy (positioning relative to components they belong to).

As for class names, I can rename `welcomeScreen__*` classes to `welcomeScreen-*` if that will make it more palatable :).